OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -452,6 +452,9 @@ public boolean isYoutubePlayerEnabled() {
             return enabled;
         }
 
+        // TODO we have removed the API key from Config as a part of
```

Comment:
auto-format needed.

Reply:
if we remove the extra spacing the todo will be converted into simple comments